pkg/startup/v14/startup_test.go

```diff
@@ -55,6 +57,34 @@ func TestFilePermissions(t *testing.T) {
 	}
 }
 
+func TestWriteResolveConf(t *testing.T) {
+	cs := &api.OpenShiftManagedCluster{
+		Properties: api.Properties{
+			NetworkProfile: api.NetworkProfile{
+				Nameservers: []string{"1.2.3.4", "5.6.7.8"},
+			},
+		},
+	}
+	tmpl := `
+{{ range .ContainerService.Properties.NetworkProfile.Nameservers }}
+nameserver {{ . }}
+{{ end }}`
```

Comment:
Is there any way to keep this in sync with the contents of `pkg/startup/v14/data/master/etc/origin/node/resolv.conf`? I'm worried that since this test is sort of unrelated they might drift. Can we read in the template from that file (or that of the current plugin) directly for the purposes of testing this?

Reply:
i'll just delete the test, it was just me confirming it was working